test/tests/mesh_modifiers/parsed_subdomain/parsed_subdomain_mm.i

```diff
@@ -0,0 +1,27 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+  xmax = 1
+  ymax = 1
+  uniform_refine = 2
+[]
+
+[MeshModifiers]
+  [./subdomains]
+    type = ParsedSubdomainMeshModifier
+    combinatorial_geometry = 'x>0.1&x<0.9&y>0.1&y<0.9'
```

Comment:
We can have spaces in ParsedFunction expressions if we use quotes, right?  Might make this a bit easier to read...